content/blog/2020-12-03-vivanco/blogpost.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Der neue Vivanco B2B Webshop'
+slug: neuer-vivanco-webshop
+templateKey: blog-post
+image: screenshot.png
+date: 2020-12-03
+author: Patrik Stutz
+description: >-
+  Nach über einem Jahr Entwicklung ist der neue B2B Webshop von Vivanco im Oktober 2020 online gegangen.
+categories:
+  - Digitale Transformation
+  - Allgemein
+---
+
+## Neuer Vivanco B2B Webshop
+
+Vivanco ist einer der führenden B2B Anbieter für Geräte- und Verbindungszubehör in Europa. Immer häufiger
+werden auch im B2B Umfeld Bestellungen online getätigt. Doch der alte Webshop war inzwischen in die Jahre gekommen und ein Weiterbetrieb bald nicht mehr möglich. Aus diesem Grund hat sich Vivanco an uns gewandt, um [einen komplett neuen Shop](https://shop.vivanco.com) zu entwickeln.
+
+### Spezifische Anforderungen
+
+Nach ersten Gesprächen war schnell klar, dass die Anforderungen zu spezifisch waren, um diesen mit einer Shop-Software von der Stange gerecht zu werden. Vivanco ist ein sehr grosses Unternehmen, mit vielen Standorten weltweit. Die einzelnen Standorte verwenden zum Teil völlig unterschiedliche Software und Geschäftsprozesse sind anders strukturiert. Dennoch muss am Ende alles in einem Shop zusammenführen.
+
+### Einführung einer PIM Software
+
+Eine ganz spezielle Herausforderung war, dass bei Vivanco noch keine zentrale & einheitliche Software zum Verwalten von Produktinfos im Einsatz war. Diese wurden vorher an unterschiedlichen Orten gepflegt. Anstelle einer Neu-Entwicklung haben wir uns hier für den Einsatz einer PIM-Software entschieden, die nun als Quelle für Produktinfos für den Webshop dient.
+
+<br/>
+
+![Produktseite](./screenshot2.png)
+
+### Zusammenarbeit mit Braincept
+
+Aus Kapazitätsgründen haben wir uns bei diesem Projekt für eine Partnerschaft mit [Braincept](https://braincept.com/) entschieden. Dabei übernahm Braincept die Entwicklung der gesamten Benutzeroberfläche des Shops, während wir uns um das Backend, das "Gehirn" des Shops gekümmert haben, welches die Daten aus vielen verschiedenen Systemen an einem Ort zusammenführt.
```

Comment:
Ich würde dies weniger betonen und einfach in einem Satz sagen, dass uns Braincept unterstützt hat. Interessanter wäre noch zu erwähnen das dieser Shop mit MS Dynamics zusammenarbeitet und wie es mit den Fotos gelöst wurde. Vor allem die Integration in das ERP ist interessant und auch für SEO von Vorteil.